runtime/tests/runtime_integration_tests/test_boot.rs

```diff
@@ -33,6 +33,13 @@ fn test_boot() {
     model.step_until_exit_success().unwrap();
 }
 
+#[test]
+fn test_persistent_data() {
```

Comment:
Would be good to comment on why this test is different from https://github.com/chipsalliance/caliptra-sw/blob/f845e85c1c3192823eae16e37ff31b0362d18efe/drivers/test-fw/src/bin/persistent_tests.rs#L9C1-L9C1. I think basically the idea is to make sure ROM/FMC didn't corrupt something in persistent data. Which I agree is important.

Reply:
Done